aiormq/base.py

```diff
@@ -137,10 +137,8 @@ async def close(
         if self.is_closed:
             return None
 
-        await asyncio.wait_for(
-            self.loop.create_task(self.__closer(exc)),
```

Comment:
Here should not have been task and `await asyncio.wait_for(self.__closer(exc), timeout)` was enough, am I right?

Reply:
Now it's doesn't matter